docs/concepts/Alexa-Hosted-Skill-Commands.md

```diff
@@ -0,0 +1,118 @@
+# CREATE SKILL - NEW COMMAND
+
+`askx new` allows developers to create an `Alexa hosted skill`, and clone the skill project to the local machine, which replaces and optimizes `ask create-hosted-skill` command in CLI V1.
+
+Alexa hosted deployer builds Alexa skills quickly by provisioning required resources automatically. 
+Alexa hosted service will create a new AWS account with a deployment pipeline (CodeBuild), a skill Lambda function, a Git repository (CodeCommit) and an S3 bucket for developers.
+Thus, the developers can then develop on the editor inside the portal or checkout the code using Git and develop with their IDEs.
+
+**STRUCTURE OF INIT COMMAND:**
+
+`askx init [-p | --profile <profile>] [--debug] [-h | --help]`
+
+**OPTIONS DESCRIPTION:**
+
+**profile**: Optional. Specify a profile name to be used. Defaults to use `default` as the profile name, if this option or environmental variable `ASK_DEFAULT_PROFILE` is not set.
+
+**debug**: Optional. Appends a debug message to the standard error.
```

Comment:
Why appends to the standard error? debug does not work if no error?

Reply:
The description is not that correct, but I refer to our official commands Reference https://developer.amazon.com/en-US/docs/alexa/smapi/ask-cli-command-reference.html#clone-command. 
I changed it to "Show debug messages."